web/docs/auth/migrate-from-11.md

```diff
@@ -86,51 +142,195 @@ You can read more about the new auth system in the [Auth Entities](./entities) s
+   mappings of old imports to new ones in [this table](https://docs.google.com/spreadsheets/d/1QW-_16KRGTOaKXx9NYUtjk6m2TQ0nUMOA74hBthTH3g/edit#gid=1725669920).
+9. Replace the Wasp file in `foo` with the Wasp file from `foo_old`
+10. Change the Wasp version field in your Wasp file (now residing in `foo`) to `"^0.12.0"`.
+11. Correct external imports in your Wasp file (now residing in `foo`).
+    imports. You can do this by running search-and-replace inside the file:
+
+    - Change all occurences of `@server` to `@src/server`
+    - Change all occurences of `@client` to `@src/client`
+
+    For example, if you previously had something like:
+
+    ```js
+    page LoginPage {
+      // highlight-next-line
+      // This previously resolved to src/client/LoginPage.js
+      // highlight-next-line
+      component: import Login from "@client/LoginPage"
+    }
+
+    // ...
+
+    query getTasks {
+      // highlight-next-line
+      // This previously resolved to src/server/queries.js
+      // highlight-next-line
+      fn: import { getTasks } from "@server/queries.js",
+    }
+    ```
+
+    You should change it to:
+
+    ```js
+    page LoginPage {
+      // highlight-next-line
+      // This now resolves to src/client/LoginPage.js
+      // highlight-next-line
+      component: import Login from "@src/client/LoginPage"
+    }
+
+    // ...
+
+    query getTasks {
+      // highlight-next-line
+      // This now resolves to src/server/queries.js
+      // highlight-next-line
+      fn: import { getTasks } from "@src/server/queries.js",
+    }
+    ```
+
+    Do this for all external imports in your `.wasp` file. After you're done, there shouldn't be any occurences of strings `"@server"` or `"@client"`
+
+12. Take all the dependencies from `app.dependencies` declaration in
+    `foo/main.wasp` and move them to `foo/package.json`. Make sure to remove the `app.dependencies` field from `foo/main.wasp`.
+
+    For example, if `foo_old/main.waps` had:
+
+    ```css
+    app Foo {
+      // ...
+      dependencies: [ ('redux', '^4.0.5'), ('reacjt-redux', '^7.1.3')];
+    }
+    ```
+
+    Your `package.json` in `foo` should now list these dependencies (Wasp already generated most of the file, you just have to list additional dependencies).
+
+    ```json
+    {
+      "name": "foo",
+      "dependencies": {
+        "wasp": "file:.wasp/out/sdk/wasp",
+        "react": "^18.2.0",
+        // highlight-next-line
+        "redux": "^4.0.5",
+        // highlight-next-line
+        "reactjs-redux": "^7.1.3"
+      },
+      "devDependencies": {
+        "typescript": "^5.1.0",
+        "vite": "^4.3.9",
+        "@types/react": "^18.0.37",
+        "prisma": "4.16.2"
+      }
+    }
+    ```
+
+13. Copy all lines you might have added to `foo_old/.gitignore` into
+    `foo/.gitignore`
+14. Copy the rest of the top-level files (excluding `gitignore` and the `.wasp`
+    file) in `foo_old/` into `foo/` (overwrite the existing files in `foo`)
```

Comment:
This step felt a bit scary to me. I wasn't sure if I might copy something I shouldn't. Would be great if you could list files that you expect me to copy, or at least give me an example.

For example, I copied .env.server, fly-client.toml, fly-server.toml, README.md, .waspignore, .wasproot.
I skipped `.gitignore` and `main.wasp`.
I also skipped dirs `.wasp` and `migrations`.
But should I have copied migrations? They are however not a top level FILE, so according to the instructions, I shouldn't copy them (while I am pretty sure I should, right? Because I know a lot about them).

Reply:
Hm, yes, I could say "files and folders" (which would include migrations).

I wouldn't list all the files because most apps don't use most of them (there's env files, tailwind, postcss, prettier, ...) and it would end up verbose and confusing.

I can change the wording if you have a suggestion.